tests/publiccloud/boottime.pm

```diff
@@ -350,7 +350,7 @@ sub store_in_db {
     my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
     return unless ($url);
     my $db = get_var('PUBLIC_CLOUD_PERF_DB', 'perf');
-    my $token = get_required_var('_PUBLIC_CLOUD_PERF_DB_TOKEN');
+    my $token = get_required_var('_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN');
```

Comment:
As told in [note above](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17229/files#r1228012006), take care of the var.value in this case, that shall be different from that other case.